ingest_data_Lynn.R

```diff
@@ -0,0 +1,236 @@
+# This script loads in the filtered data and checks for invalid rows, as well as changes the
+# neighborhood names to be compatible with the neighborhoods table in the data base.
+# It then inserts the data into the data base.
+
+# General setup-----
+
+library(RPostgreSQL)
+con <- dbConnect(PostgreSQL(), user="lhk", password = "0ntBNyCcuipdh8g", 
+                 dbname="lhk", host="pg.stat.cmu.edu")
+
+# Change the schema that I will access with the connection
+set.schema <- dbSendStatement(con, "set search_path to 'schemacrime';")
+
+library(testthat)
```

Comment:
Oh, sorry, I missed that. That's a slightly unusual way of doing things -- it works, since `expect_false` raises a condition when it fails, but it's meant for use inside `test_that` calls. You could equivalently do `if (some condition is false) { print appropriate error message }` or code to that effect.